packages/core/src/styles/tokens/tokens.stories.ts

```diff
@@ -25,36 +32,21 @@ interface Token {
 
 function getTokenTable(key: string, globals: any = {}) {
   return html` <style>
-      .token-table {
-        width: 100%;
-      }
-
-      .token-table tr {
-        border-bottom: 1px solid var(--cds-alias-object-border-color-tint);
-        line-height: 0;
-        min-height: 45px;
-        width: 100%;
-      }
-
-      .token-table tr:last-child {
-        border-bottom: 0;
-      }
-
-      .token-table td {
-        padding: 0;
+      .token-table td > div {
+        line-height: 1em;
       }
     </style>
-    <table cds-table="b:row b:outside p:lg">
+    <table class="token-table" cds-table="border:row border:outside" cds-text="left">
```

Comment:
Tokens table was broken from previous PR

Reply:
Outside of the scope of this PR... but do you think regression testing would catch regressions like this?

Reply:
I think visual snapshots fo the stories probably would. This was broken in the original table PR when it was merged. I didn't catch it in review.

Reply:
Noted that visual regression effort is currently in progress for the core storybook.